### PR TITLE
Add CZI as an Institutional and Funding Partner

### PR DESCRIPTION
**Pull request by sofroniewn**
# Description
Following NAP-1 added in #4458 and discussion of the @napari/steering-council we will be adding CZI as napari's first institutional and funding partner - and with agreement of the necessary CZI leadership I @sofroniewn will be it's IFP representative - cc @neuromusic @justinelarsen 

We will transition my seat on the napari steering council to be the first IFP seat, I will become the first holder of the IFP seat and first member of the IFP advisory council.

In the immediate sense this means nothing is changing, but in the longer term it might lead to the following changes. For example if the CZI Leadership were to feel I was no longer representing CZI's interests or if I were to leave CZI then CZI could replace me on the IFP Advisory Council with a new representative - triggering a process of redetermining the holder of the IFP seat on the SC. If the napari SC is still only three people, then at this point, the napari SC could nominate me to hold a seat on the SC again if the project felt it was in its best interests - but that would be their choice. I would retain my core developer status if I was still active in the project.

As the first member of the IFP AC I promise to work in good faith with subsequent members of the IFP AC to determine the process for selecting the IFP AC representative on the SC. I think we can wait until new IFPs are seeking to join napari before determining any rules, but I imagine that some form of vote weighting proportional to the level of investment of the IFP would be appropriate.

Please let me know any concerns or feedback - thanks!!

### CODE REVIEW

**Review by jni — APPROVED** [review]
👍👍👍

File: docs/community/team.md
Comment (alisterburt) [inline docs/community/team.md]: inconsistent, Imaging Tech vs Imaging Technology

**Review by alisterburt — APPROVED** [review]
🎉

File: docs/community/team.md
Comment (sofroniewn) [reply docs/community/team.md]: fixed, thanks!

**Review by kevinyamauchi — APPROVED** [review]
This looks good to me. Thanks, everyone!

**Comment by tlambert03** [comment]
> For example if the CZI Leadership were to feel I was no longer representing CZI's interests or if I were to leave CZI then CZI could replace me on the IFP Advisory Council with a new representative - triggering a process of redetermining the holder of the IFP seat on the SC.
> As the first member of the IFP AC I promise to work in good faith with subsequent members of the IFP AC to determine the process for selecting the IFP AC representative on the SC. I think we can wait until new IFPs are seeking to join napari before determining any rules, but I imagine that some form of vote weighting proportional to the level of investment of the IFP would be appropriate.

One question I had, (this is probably a NAP-1 question, but I couldn't  find anything explicitly mentioned there) is about the nature of this transition.  If you were to leave, who decides who fills your seat?  From the wording, I'm gathering that the IFP itself picks. Or perhaps the steering council needs to approve the IFP pick (which, if it were to happen now, would essentially be a 2 person vote).  I would be interested to have that process detailed a little more clearly.  "Good faith" is all fine and good, but pretty much anything falls under good faith :joy:
Obviously, I am thrilled to be "steered" by @sofroniewn, given his tremendous dedication and intimate knowledge of napari itself and the community it serves.  But I can imagine a world in which someone with much less understanding about napari and its community comes on to the steering council... and then holds significantly more influence than many core developers who have spent years building the thing.  

If it is the case that the core dev team definitely _doesn't_ have a direct say in how IFPs decide their SC representation, it would be nice to know that for sure.

**Comment by sofroniewn** [comment]
> From the wording, I'm gathering that the IFP itself picks. Or perhaps the steering council needs to approve the IFP pick (which, if it were to happen now, would essentially be a 2 person vote)

> If it is the case that the core dev team definitely doesn't have a direct say in how IFPs decide their SC representation, it would be nice to know that for sure.

Yes, the IFP itself gets to pick their representative for the IFP advisory council - the SC does not get a vote on that person and neither does the core developer team. All the members of the IFP advisory council (one from each IFP) would then get to vote again on who they'd like represent them on the SC using the rules they have agreed on. The SC nor core devs would not get to overrule that vote - unless there was some egregious violation of the good faith stipulation - for example, voting someone that had received multiple code of conduct violations in napari.

We can update the discussion section of the NAP to make that more explicit - cc @jni 

> But I can imagine a world in which someone with much less understanding about napari and its community comes on to the steering council... and then holds significantly more influence than many core developers who have spent years building the thing.

Yes - that could happen, but they would be at most one voice out of three or five people on the SC, and might collectively be representing the funding that enables the sustainability and growth of the project - for example, by supporting those other core developers. The rationale for this is that it incentivizes institutional collaboration that can provide such resources, while ensuring that funder concerns don't dominate the community. With votes decided by a simple majority, the IFP seat alone cannot block or derail the project. We were inspired here in part by the governance of https://openforcefield.org/about/organization/ and learnings from the funding of open source software through CZI's EOSS program. I will acknowledge that it is an experiment and there isn't one guaranteed way to do governance, and other projects have structured their governance differently. This structure is an attempt to learn from conversations with many projects and put something in place that will help napari for the long-term.

**Comment by liu-ziyang** [comment]
Kinda above my paycheck but just offering my 2 cents here: I think the IFP introduced a nice model of offering the interested party a collective voice in the project. As the project grows and matures, I would imagine we may similarly introduce voice from the plugin developer community, and our user community, each may elect their own representative (I don't know how at the moment) so that their voice is heard and their vote counted, and the steering council then would really be representing everyone involved in this project's success. (maybe core developers will elect their representative in the SC similarly to another interested party, instead of being the sole steering force other than IFP, but just with more seats at that point).

**Comment by tlambert03** [comment]
> Yes, the IFP itself gets to pick their representative for the IFP advisory council - the SC does not get a vote on that person and neither does the core developer team. We can update the discussion section of the NAP to make that more explicit - cc @jni

thanks, yes I'd like to see that made clear

> but they would be at most one voice out of three or five people on the SC

one out of 3 is quite a bit!  it's (obviously) one vote away from a majority.

**Comment by kne42** [comment]
i think it would be better to have the IFP rep to be explicitly used as a tiebreaker vote (they essentially will always act as this given that the SC will always be odd-membered and they can only hold 1 seat)

**Comment by sofroniewn** [comment]
> I would imagine we may similarly introduce voice from the plugin developer community, and our user community, each may elect their own representative (I don't know how at the moment) so that their voice is heard and their vote counted, and the steering council then would really be representing everyone involved in this project's success. 

These are very interesting ideas, and maybe become the subject of future NAPs if in the projects best interest

> i think it would be better to have the IFP rep to be explicitly used as a tiebreaker vote (they essentially will always act as this given that the SC will always be odd-membered and they can only hold 1 seat)

As you point out this is essentially true by our rules, and so maybe is a helpful way to mentally view that seat right now

> thanks, yes I'd like to see that made clear

Sure, i will update the NAP1 discussion with all these points when I get the chance

**Pull request merged**
